addons/hi-res-thumbnails/addon.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Higher resolution thumbnails",
```

Comment:
I decided on naming it "Higher resolution thumbnails" instead of "High resolution thumbnails" for a sort of cheeky reason: this way, when you read it, you wonder, "how *much* higher?"—and when you click to find out, it turns out the answer is you get to choose!

The default (200% width & height) is still a sensible value for anyone on a retina screen. If you're the sort of person who browses at a higher than normal zoom level, you probably already understand the specifics of what makes images blurry when you're zoomed in, and will know to set the value to however much you're normally zoomed.

Reply:
yeah, the thing is, now it's a verb, and we want to try and keep addon names as nouns wherever possible